README.md

```diff
@@ -38,7 +38,7 @@ Full documentation can be found at <https://gp2040-ce.info>
 * Left and Right stick emulation via D-pad inputs as well as dedicated toggle switches.
 * Dual direction via D-pad + LS/RS.
 * Reversed input via a button.
-* Turbo and Turbo LED with selectable speed
```

Comment:
(test comment)

Reply:
The addition of the Github link is unnecessary as there is already a corner icon that links to the Github repo.

![image](https://github.com/OpenStickCommunity/GP2040-CE/assets/44279698/8fef593b-161b-44c9-9db4-785d2d7e89ef)

Reply:
I see. I actually never noticed that corner icon, it's kind of easy to miss. I think it might still be worth having an extra link for it, just for safety's sake. There's already duplicated links from https://gp2040-ce.info to itself